packages/nextjs/app/builders/[0xC4Aad525854Cc8d21122Bee8CcC1d9c3cEcBb859]/page.tsx

```diff
@@ -0,0 +1,100 @@
+"use client";
+
+import { useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import type { NextPage } from "next";
+import { FaGithub, FaLinkedin, FaTwitter } from "react-icons/fa";
+import { Address } from "~~/components/scaffold-eth";
+
+const BuilderProfile: NextPage = () => {
+  const address = "0xC4Aad525854Cc8d21122Bee8CcC1d9c3cEcBb859";
+  // const shortenedAddress = address?.slice(0, 6) + "..." + address?.slice(-4);
```

Comment:
It's bad practice to leave commented code. Other contributors might be confused about whether it should be uncommented or removed.